internals-js/src/federation.ts

```diff
@@ -1315,28 +1297,15 @@ export class FederationMetadata {
       baseDirectives.push(policyDirective);
     }
 
-    const sourceAPIDirective = this.sourceAPIDirective();
-    if (isFederationDirectiveDefinedInSchema(sourceAPIDirective)) {
-      baseDirectives.push(sourceAPIDirective);
-    }
-    const sourceTypeDirective = this.sourceTypeDirective();
-    if (isFederationDirectiveDefinedInSchema(sourceTypeDirective)) {
-      baseDirectives.push(sourceTypeDirective);
-    }
-    const sourceFieldDirective = this.sourceFieldDirective();
-    if (isFederationDirectiveDefinedInSchema(sourceFieldDirective)) {
-      baseDirectives.push(sourceFieldDirective);
-    }
-
-    const contextDirective = this.contextDirective();
-    if (isFederationDirectiveDefinedInSchema(contextDirective)) {
-      baseDirectives.push(contextDirective);
-    }
+      const contextDirective = this.contextDirective();
+      if (isFederationDirectiveDefinedInSchema(contextDirective)) {
+          baseDirectives.push(contextDirective);
+      }
```

Comment:
```suggestion
    const contextDirective = this.contextDirective();
    if (isFederationDirectiveDefinedInSchema(contextDirective)) {
      baseDirectives.push(contextDirective);
    }
```
Looks like 4 space tabs snuck in here